src/cbc_sdk/live_response_api.py

```diff
@@ -1032,6 +1210,8 @@ def __init__(self, cb, timeout=30, keepalive_sessions=False):
         self._keepalive_sessions = keepalive_sessions
         self._init_poll_delay = 1
         self._init_poll_timeout = 360
+        self._async_executor = None
+        self._thread_pool_count = self.THREAD_POOL_COUNT
```

Comment:
This should be configurable in the `__init__` function

Reply:
Added as optional parameter.